scripts/testing/install_dependencies.sh

```diff
@@ -42,7 +42,9 @@ sed 's/@PACKAGE_VERSION@/0/; s/@PACKAGE_RELEASE@/0/; s/%{__python3}/python3/' ./
 build_deps=$(rpmspec -q --buildrequires $TEMP | sed 's/>=.*$//')
 # add also runtime dependencies for the local development
 # remove anaconda packages and also '(glibc-langpack-en or glibc-all-langpacks)' which will fail otherwise
-requires_deps=$(rpmspec -q --requires $TEMP | grep -v -E "(anaconda-|-widgets| or )" | sed 's/>=.*$//')
+# remove podman, ostree (and skopeo as ostree dependency) because these are not required for development of Anaconda
+# and are breaking easy use of host commands from the container (distrobox-host-exec)
```

Comment:
You can do something like:

`rpmspec --define "build_bootc  0 ..."

Reply:
And in the spec file:

``` diff
diff --git anaconda.spec.in anaconda.spec.in
index 709ab7fcc2..0602d95d42 100644
--- anaconda.spec.in
+++ anaconda.spec.in
@@ -46,6 +46,7 @@ Source0: https://github.com/rhinstaller/%{name}/releases/download/%{name}-%{vers
 %define subscriptionmanagerver 1.26
 %define utillinuxver 2.15.1
 %define rpmostreever 2023.2
+%define build_bootc 1
 
 BuildRequires: libtool
 BuildRequires: gettext-devel >= %{gettextver}
@@ -273,7 +274,9 @@ Requires: skopeo
 # External tooling for managing NVMe-FC devices in the installation environment
 Requires: nvme-cli
 # Needed for bootc
+%if %{build_bootc}
 Requires: podman
+%endif
```

Reply:
This is great. I wasn't aware that `rpmspec` supports this. Sounds good.